### PR TITLE
Lesson 7, task 1 'Product card' - Done!

### PR DESCRIPTION
**Pull request by Atteroot**
- [DEMO LINK](https://Atteroot.github.io/layout_product-cards/) <br>
- [TEST REPORT LINK](https://Atteroot.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  margin-top: 15px;
+  margin-left: 5px;
+  margin-bottom: 42px;
```
Comment (danheim) [inline src/style.css:35]: `margin: ...;`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code-name {
+  line-height: 13px;
+  margin-top: 5px;
```
Comment (danheim) [inline src/style.css:46]: add margin-bottom to element above

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.card__rating {
+  margin-top: 15px;
```
Comment (danheim) [inline src/style.css:50]: same

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  background-image: url(/images/star-active.svg);
+}
+
+.card__votes {
+  line-height: 13px;
+  margin-top: 1px;
+}
+
+.card__price {
+  margin-top: 20px;
```
Comment (danheim) [inline src/style.css:80]: same

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  margin-top: 3px;
+}
+
+.price__value {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.button {
```
Comment (danheim) [inline src/style.css:96]: add cursor pointer

**Review by danheim — CHANGES_REQUESTED** [review]
use margin-bottom instead of margin-top

File: src/style.css (hunk 1 above)
Comment (Atteroot) [reply src/style.css:35]: Fixed!

File: src/style.css (hunk 2 above)
Comment (Atteroot) [reply src/style.css:46]: Fixed!

File: src/style.css (hunk 3 above)
Comment (Atteroot) [reply src/style.css:50]: Fixed!

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+.stars--2 .stars_star:nth-child(-n+2),
+.stars--3 .stars_star:nth-child(-n+3),
+.stars--4 .stars_star:nth-child(-n+4),
+.stars--5 .stars_star:nth-child(n) {
+  background-image: url(/images/star-active.svg);
+}
+
+.card__votes {
+  line-height: 13px;
+  margin-top: 1px;
```
[the remark this answers is not in the document]
Comment (Atteroot) [reply src/style.css:76]: Лишнее, удалил совсем

File: src/style.css (hunk 4 above)
Comment (Atteroot) [reply src/style.css:80]: Fixed!

File: src/style.css (hunk 5 above)
Comment (Atteroot) [reply src/style.css:96]: Added!

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**